README.md

```diff
@@ -233,6 +234,26 @@ https://github.com/openresty/lua-nginx-module/blob/master/.travis.yml
 
 [Back to TOC](#table-of-contents)
 
+reindex
+---------
+
+The `reindex` is used to unify the block title format of the test files, which are written based on
+[Test::Base](https://metacpan.org/pod/Test::Base). And you can install this tool like
+[ngx-build](#ngx-build).
+
+The `reindex` will unify the block title format of each test file according to the following conditions:
+* Ensure that all test blocks starting with `=== TEST {$index}` are sequentially numbered starting from `${begin_index}`.
+* Ensure 3 newlines between two blocks.
+* Ensure that there is a line break between the first test case and the separator(`__DATA__` or `__END__`).
+
+You can run `reindex` like this:
+```
+# -b: the begin index of test blocks in each test file.
+reindex -b 1 /path/to/module/t*.t
```

Comment:
the `-b 1` is the default option, we can omit here?

Reply:
> the `-b 1` is the default option, we can omit here?

Updated, is it ok like this: https://github.com/openresty/openresty-devel-utils/pull/29/files#diff-04c6e90faac2675aa89e2176d2eec7d8R249